smurfs/src/actions/index.js

```diff
@@ -13,3 +13,40 @@
    U - updateSmurf
    D - deleteSmurf
 */
+import axios from 'axios';
+
+export const FETCHING = 'FETCHING';
+export const SUCCESS = 'SUCCESS' ;
+export const FAILURE = 'FAILURE' ;
+export const ADDING = 'ADDING'  ;
+export const DELETING = 'DELETING';
+
+export const getSmurfs = () => (dispatch) => {
+        dispatch  ({type: FETCHING})
+    axios
+          .get('http://localhost:3333/smurfs')
+          .then(res => {
```

Comment:
You can also declare the url as a constant and then if it ever changes you only need to change it in one spot